app/src/main/kotlin/com/wire/android/ui/home/conversations/media/preview/AssetFilePreview.kt

```diff
@@ -68,36 +68,24 @@ fun AssetFilePreview(
             Text(
                 modifier = Modifier.padding(bottom = dimensions().spacing8x),
                 text = assetName.split(".").last().uppercase(Locale.getDefault()),
-                style = MaterialTheme.wireTypography.title01.copy(
-                    fontWeight = FontWeight.W900,
-                    color = MaterialTheme.colorScheme.onPrimary
-                )
+                color = MaterialTheme.colorScheme.onPrimary,
+                style = MaterialTheme.wireTypography.title05
             )
         }
         VerticalSpace.x16()
         Text(
             assetName,
-            style = MaterialTheme.wireTypography.title02.copy(color = MaterialTheme.colorScheme.onBackground),
+            style = MaterialTheme.wireTypography.title02,
+            color = MaterialTheme.colorScheme.onBackground,
             textAlign = TextAlign.Center,
             maxLines = 2,
             overflow = TextOverflow.Ellipsis
         )
         VerticalSpace.x8()
-        Text(sizeInBytes.toFileSize(), style = MaterialTheme.wireTypography.body01.copy(MaterialTheme.wireColorScheme.secondaryText))
-    }
-}
-
-@Suppress("MagicNumber")
-private fun Long.toFileSize(): String {
-    val kilobyte = 1024.0
-    val megabyte = kilobyte * 1024
-    val gigabyte = megabyte * 1024
-
-    return when {
-        this < kilobyte -> "$this B"
-        this < megabyte -> String.format("%.2f KB", this / kilobyte)
-        this < gigabyte -> String.format("%.2f MB", this / megabyte)
-        else -> String.format("%.2f GB", this / gigabyte)
+        Text(
+            DeviceUtil.formatSize(sizeInBytes),
+            style = MaterialTheme.wireTypography.body01.copy(MaterialTheme.wireColorScheme.secondaryText)
```

Comment:
Can do here the same as above, move the color to `color = ..` instead of inside the `style`